logging/interceptor_test.go

```diff
@@ -38,7 +39,7 @@ var (
 	buf         bytes.Buffer
 	reader      io.Reader
 	testLogger  = New("Info")
-	testMD      = metautils.NiceMD{}.Set(testAuthorizationHeader, testJWT).Set(DefaultRequestIDKey, testRequestID).Set(testCustomHeaderKey, testCustomHeaderVal)
+	testMD      = metautils.NiceMD{}.Set(testAuthorizationHeader, testJWT).Set(requestid.DefaultRequestIDKey, testRequestID).Set(testCustomHeaderKey, testCustomHeaderVal)
```

Comment:
Please add a case that looks for `DeprecatedRequestIDKey` so we can verify backwards compatibility.

Reply:
Good point, did it